modules/table/src/integrationTest/java/org/apache/ignite/distributed/ItInternalTableScanTest.java

```diff
@@ -527,7 +543,7 @@ public void onSubscribe(Subscription subscription) {
             public void onNext(BinaryRow item) {
                 retrievedItems.add(item);
 
-                if (retrievedItems.size() % reqAmount == 0) {
+                if (reqAmount != Long.MAX_VALUE && retrievedItems.size() % reqAmount == 0) {
```

Comment:
What for additional  condition `reqAmount != Long.MAX_VALUE` is added?
Is the condition `retrievedItems.size() % reqAmount` **true** in this case?